build/helper/documentation_helper.py

```diff
@@ -352,7 +352,10 @@ def _replace_attribute_python_name(a_match):
             aname = attr['name'].lower()
 
     if config['make_link']:
-        return ':py:data:`{0}.Session.{1}`'.format(config['module_name'], aname)
+        if config['module_name'] == 'nitclk':
```

Comment:
How? What can we key on instead?

Reply:
If driver config had:

* Name of the main class
* Add properties to main class
* Add functions as methods to main class

Could we then make better reuse of the templates and get rid of this conditional logic?

Reply:
Yes, but to quote a top notch engineer from a different review "The medicine is worse than the disease".

It would remove using 'nitclk' in the conditionals here (but not remove them), and `session.py.mako` would become _much_ more complicated and less easy to follow with the conditionals there surrounding large blocks of code.

And I don't think it would be enough to combine the nitclk session.py.mako with the generic one. Or if it is, it would become even more complicated.

Reply:
Feels very wrong and hacky, but I can see it being the pragmatic solution. At least we tried.